.github/workflows/go.yml

```diff
@@ -26,3 +26,91 @@ jobs:
         run: make setup
       - name: Test
         run: make test
+  build_and_deploy_podium:
+    needs:
+      - tests
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/')
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set env
+        run: echo "VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - name: Generate executable
+        run: make setup cross
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build and push tag
+        id: docker_build_tag
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: tfgco/podium:${{ env.VERSION }}
+      - name: Build and push latest
+        id: docker_build_latest
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: tfgco/podium:latest
+  build_and_deploy_podium_dev:
+    runs-on: ubuntu-latest
+    needs:
+      - tests
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set env
+        run: echo "VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
```

Comment:
Shouldn't we have a suffix indicating this version is for development? If production and development images follow the same version naming convention, it is possible for someone to confuse them and deploy the wrong one. Especially if we consider that we want to allow the game devs to update their own instances. This could be confusing for them and it could lead to development images being deployed to production.